### DISCUSSION TITLE
Error while running "caffeinate -dim ./install.sh"

### DISCUSSION THREAD

**Discussion by andsanv** [ichi4096/vivado-on-silicon-mac#31] [General]
followed all steps needed and marked as essential.
en running the command, this output is shown:

<>@MacBook-Air-di-<> vivado-on-silicon-mac-main % sudo caffeinate -dim ./install.sh
Launching Docker daemon and XQuartz...
Building Docker image
[+] Building 0.8s (2/3)                                    docker:desktop-linux
[+] Building 0.9s (3/3) FINISHED                           docker:desktop-linux
 => [internal] load .dockerignore                                          0.0s
 => => transferring context: 2B                                            0.0s
 => [internal] load build definition from Dockerfile                       0.0s
 => => transferring dockerfile: 2.02kB                                     0.0s
 => **ERROR [internal] load metadata for docker.io/library/ubuntu:latest     0.9st**
<raw>------</raw>
 <raw>></raw> [internal] load metadata for docker.io/library/ubuntu:latest:
<raw>------</raw>
Dockerfile:3
<raw>--------------------</raw>
   1 |     # Container for running Vivado on M1/M2 macs
   2 |     # though it should work equally on Intel macs
   3 | >>> FROM --platform=linux/amd64 ubuntu
   4 |     RUN apt update && apt upgrade -y
   5 |     
<raw>--------------------</raw>
**ERROR: failed to solve: ubuntu: error getting credentials - err: exit status 1, out: ``**

first error appeared. then "install.sh" file execution goes on and asks for the binary file in input, which i give:

Drag and drop the installation binary into this terminal window and press Enter: 
/Users/*/Downloads/FPGAs_AdaptiveSoCs_Unified_2023.2_1013_2256_Lin64.bin 
Launching Docker container and installation script
Unable to find image 'x64-linux:latest' locally
2023/12/13 16:42:12 must use ASL logging (which requires CGO) if running as root
docker: Error response from daemon: pull access denied for x64-linux, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
Generating App icon
./install.sh:68: no matches found: Xilinx/Vivado/*/doc/images/vivado_logo.png
Warning:  not a valid file - skipping
Warning:  not a valid file - skipping
Warning:  not a valid file - skipping
Warning:  not a valid file - skipping
Warning:  not a valid file - skipping
Warning:  not a valid file - skipping
Warning:  not a valid file - skipping
Warning:  not a valid file - skipping
Warning:  not a valid file - skipping
icon.iconset:Failed to generate ICNS.
mv: rename icon.icns to Launch_Vivado.app/Contents/Resources/icon.icns: No such file or directory

probably the second error is caused by the first one, which i cannot find a way to solve.
anyone has got any idea?

**Comment by ichi4096** [comment]
The scripts have been updated, try the new version.
